script/delete_weird_company.rb

```diff
@@ -0,0 +1,32 @@
+require File.expand_path('../../config/environment', __FILE__)
+
+weird_company_wql = {
+  'not' => { 'type' => 'Company' },
+  'left_plus' => [
+    { 'type' => 'Metric' },
+    { 'right_plus' => [{ 'type' => 'Year' }, { 'type' => 'Metric Value' }] }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.